Makefile

```diff
@@ -38,6 +38,9 @@ lint_terraform: dev
 lint_shellcheck:
 	find . -name '*.sh' -not -path './vendor/*' | xargs $(SHELLCHECK)
 
+lint_concourse:
+	cd .. && python paas-cf/concourse/scripts/pipecleaner.py paas-cf/concourse/pipelines/*.yml
```

Comment:
Where we import external pipeline configuration [from other files](https://github.com/alphagov/paas-cf/blob/master/concourse/pipelines/create-bosh-cloudfoundry.yml#L1145) we refer to the files by their full filepath including the `paas-cf/` component. I _could_ have attempted to determine the root you're in and automatically stripped it, but it seemed cleaner to just switch to the location that concourse will actually run from.

Ugly, but probably easier to understand when it goes wrong.